home/pom.xml

```diff
@@ -69,13 +69,13 @@
         <dependency>
             <groupId>org.vivoweb</groupId>
             <artifactId>jena2tools</artifactId>
-            <version>1.2.0-SNAPSHOT</version>
+            <version>1.2.0</version>
```

Comment:
I believe the release plugin requires the version to be at X.Y.Z-SNAPSHOT as a starting point, no?

Reply:
:+1: